processing/src/main/java/io/druid/segment/data/GenericIndexed.java

```diff
@@ -57,7 +56,9 @@
  * bytes 7-10 =>; numElements
  * bytes 10-((numElements * 4) + 10): integers representing *end* offsets of byte serialized values
  * bytes ((numElements * 4) + 10)-(numBytesUsed + 2): 4-byte integer representing length of value, followed by bytes
```

Comment:
", followed by bytes for value" is not part of "This field has no meaning..."

Reply:
changed.